.buildkite/scripts/publish_docker_studio.ps1

```diff
@@ -29,14 +32,24 @@ if(is_fake_release) {
   try {
     Write-Host "Pushing ${docker_image}:$docker_base_image_tag-$docker_image_version"
     docker push "${docker_image}:$docker_base_image_tag-$docker_image_version"
+    if ($LASTEXITCODE -ne 0) {
+      Write-Error "Failed to upload image"
+    }
     Write-Host "Pushing ${docker_image}:$docker_base_image_tag-$docker_image_short_version tag for $docker_image_version"
     docker push "${docker_image}:$docker_base_image_tag-$docker_image_short_version"
+    if ($LASTEXITCODE -ne 0) {
+      Write-Error "Failed to tag image"
+    }
     if($docker_base_image_tag -eq "ltsc2016") {
       Write-Host "Pushing latest tag for $docker_image_version"
       docker push "${docker_image}:latest"
+      if ($LASTEXITCODE -ne 0) {
+        Write-Error "Failed to tag image"
```

Comment:
Would it be worth differentiating this error message from the one on line 41, just to give us a little extra context if something fails?

Outputting which tagging failed would be nice.

Reply:
I think the `Write-Host` should give that context

Reply:
Derp... so it does!

Ignore me :sweat_smile: